universities/organizations.json

```diff
@@ -1,492 +1,492 @@
     "country": "POL"
   },
   {
-    "id": "U-290",
+    "id": "290",
     "name": "CAU Kiel",
     "formal_name": "Christian-Albrechts-Universitaet zu Kiel",
     "country": "DEU"
   },
   {
-    "id": "U-362",
+    "id": "362",
     "name": "Darmstadt UT",
     "formal_name": "Darmstadt University of Technology",
     "country": "DEU"
   },
   {
-    "id": "U-367",
+    "id": "367",
     "name": "TU Delft",
     "formal_name": "Delft University of Technology",
     "country": "NLD"
   },
   {
-    "id": "U-452",
+    "id": "452",
     "name": "TU/e",
     "formal_name": "Eindhoven University of Technology",
     "country": "NLD"
   },
   {
-    "id": "U-573",
+    "id": "573",
     "name": "FAU",
     "formal_name": "Friedrich-Alexander-University Erlangen-Nuremberg",
     "country": "DEU"
   },
   {
-    "id": "U-980",
+    "id": "980",
     "name": "KTU",
     "formal_name": "Kaunas University of Technology",
     "country": "LTU"
   },
   {
-    "id": "U-1039",
+    "id": "1039",
     "name": "KTH",
     "formal_name": "KTH - Royal Institute of Technology",
     "country": "SWE"
   },
   {
-    "id": "U-1088",
+    "id": "1088",
     "name": "Leiden University",
     "formal_name": "Leiden University",
     "country": "NLD"
   },
   {
-    "id": "U-1097",
+    "id": "1097",
     "name": "LIU",
     "formal_name": "Linköping University",
     "country": "SWE"
   },
   {
-    "id": "U-1112",
+    "id": "1112",
     "name": "LU",
     "formal_name": "Lund University",
     "country": "SWE"
   },
   {
-    "id": "U-1154",
+    "id": "1154",
     "name": "Uni Lübeck",
     "formal_name": "Universität zu Lübeck",
     "country": "DEU"
   },
   {
-    "id": "U-1362",
+    "id": "1362",
     "name": "Norwegian TNU",
     "formal_name": "Norwegian University of Science and Technology",
     "country": "NOR"
   },
   {
-    "id": "U-1534",
+    "id": "1534",
     "name": "RU Groningen",
     "formal_name": "Rijksuniversiteit Groningen",
     "country": "NLD"
   },
   {
-    "id": "U-1917",
+    "id": "1917",
     "name": "TUM",
     "formal_name": "Technische Universität München",
     "country": "DEU"
   },
   {
-    "id": "U-2061",
+    "id": "2061",
     "name": "UmU",
     "formal_name": "Umeå University",
     "country": "SWE"
   },
   {
-    "id": "U-2267",
+    "id": "2267",
     "name": "UdS",
     "formal_name": "Saarland University",
     "country": "DEU"
   },
   {
-    "id": "U-2272",
+    "id": "2272",
     "name": "KIT",
     "formal_name": "Karlsruhe Institute of Technology",
     "country": "DEU"
   },
   {
-    "id": "U-2274",
+    "id": "2274",
     "name": "Uni Rostock",
     "formal_name": "Universität Rostock",
     "country": "DEU"
   },
   {
-    "id": "U-2275",
+    "id": "2275",
     "name": "Ulm",
     "formal_name": "Universität Ulm",
     "country": "DEU"
   },
   {
-    "id": "U-2282",
+    "id": "2282",
     "name": "Leiden University",
     "formal_name": "Leiden University",
     "country": "NLD"
   },
```

Comment:
Possibly unrelated, but duplicate of `1088`

Reply:
yes, this is one of the duplicates in ICPC